contracts/rollup/Rollup.sol

```diff
@@ -173,6 +173,10 @@ contract Rollup is BatchManager, EIP712, IEIP712 {
             );
     }
 
+    function validateBatchID(uint256 batchID) internal view {
```

Comment:
1. We can make `validateBatchID` a modifier so that it is consistent with other `require` checks like `onlyCoordinator` and `isNotRollingBack`.
2. Suggest documenting the reasons for this check or referring to #600, so that future auditor/reviewer can understand this better.